schemas/2018-05-01/subscriptionDeploymentTemplate.json

```diff
@@ -649,18 +649,32 @@
     "output": {
       "type": "object",
       "properties": {
+        "condition": {
+          "oneOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "Condition of the output"
+        },
         "type": {
           "$ref": "#/definitions/parameterTypes",
           "description": "Type of output value"
         },
         "value": {
           "$ref": "#/definitions/parameterValueTypes",
           "description": "Value assigned for output"
+        },
+        "copy": {
+          "$ref": "#/definitions/outputCopy",
+          "description": "Output copy"
         }
       },
       "required": [
-        "type",
-        "value"
+        "type"
```

Comment:
just curious, is value no longer required?

Reply:
It's not required when using "copy" - see [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-outputs?tabs=azure-powershell#dynamic-number-of-outputs) for an example.